### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by tylervalentine**
What we have so far:
- Removed mongo collection and work server doc files
- Updated production doc file descriptions of various packages throughout the project
- Removed mention of work server in the client doc file

What still needs to be done:
- Remove (or update?) EC2-Mongo doc file
- Mention attachments in API doc file
- Remove endpoints relating to the work server in client doc file, and possibly document any important functions

### CODE REVIEW

**Comment by tylervalentine** [comment]
This PR is ready for review.

File: docs/api.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ https://api.regulations.gov/v4/END_POINT?filter[lastModifiedDate][ge]=2020-08-10
 ### **What the client should do**
 While the above section describes how to generate all jobs, the client will do the actual work for each job. 
 
-This will be relatively simple: return the JSON given by the endpoint to the work server.
+The client will get a job from regulations.gov, process the job URL, and finally downloads the job.
 
-Additionally, if a document/comment is being retrieved, return any attachments that go along with that document/comment. These can be identified in the JSON given under data/attributes/fileFormats.
+Additionally, if a document/comment is being retrieved, it will return any attachments that go along with that document/comment. These can be identified in the JSON given under data/attributes/fileFormats.
```
Comment (justinszaro) [inline docs/api.md:44]: You may want to include that the client will also turn around and download these attachments.

File: docs/client.md [hunk 2]
```diff
@@ -2,21 +2,11 @@
 
 
 ## Summary
-The clients are their own objects that will connect to the work server, receive work, 
-and send results back to the server by making the calls to [regulations.gov]
-(https://www.regulations.gov/) for data downloads. 
+The clients are their own objects that will request work from the job queue, perform the work by making calls to [regulations.gov]
```
Comment (justinszaro) [inline docs/client.md:5]: "and perform the work by making calls..."

File: docs/production.md [hunk 3]
```diff
@@ -25,3 +24,5 @@ The system is Dockerized into a number of components:
 * `docker compose down` to bring the system down
```
Comment (justinszaro) [inline docs/production.md:24]: Could you add the --tail=## tag to the documentation as well? It may be helpful for the next class to know.

**Review by justinszaro — CHANGES_REQUESTED** [review]

File: docs/production.md [hunk 4]
```diff
@@ -8,20 +8,22 @@ The system is Dockerized into a number of components:
...
+* `client1` through `client25` - Clients that download data from regulations.gov
 
 ## Docker Compose commands
 
 * `docker compose build` to build containers
 * `docker compose up -d` to start all containers (`-d` to run in background)
-* `docker compose logs` to see logs.  Optional, add a container name to see
-  individual logs
+* `docker compose logs` to see logs.  
+	* `--tail <number>` add to display a specific number of logs
```
Comment (justinszaro) [inline docs/production.md:23]: The tag is --tail=<number>

**Review by justinszaro — CHANGES_REQUESTED** [review]

**Review by justinszaro — APPROVED** [review]

**Pull request merged**